jumpcloud-ADMU/Powershell/Public/Start-Reversion.ps1

```diff
@@ -0,0 +1,418 @@
+                    # This informs Windows where to find the restored .DAT files
+                    Write-ToLog -Message "Updating ProfileImagePath from '$registryProfileImagePath' to '$profileImagePath'" -Level Verbose -Step "Revert-Migration"
+
+                    Set-ItemProperty -Path $profileRegistryPath -Name "ProfileImagePath" -Value $profileImagePath -Force
+
+                    # Verify the update was successful
+                    $updatedPath = (Get-ItemProperty -Path $profileRegistryPath -Name "ProfileImagePath").ProfileImagePath
+                    if ($updatedPath -eq $profileImagePath) {
+                        Write-ToLog -Message "Successfully updated registry ProfileImagePath to: $profileImagePath" -Level Info -Step "Revert-Migration"
+                        $revertResult.RegistryUpdated = $true
+                    } else {
+                        throw "Registry update verification failed. Expected: '$profileImagePath', Got: '$updatedPath'"
+                    }
+                }
+            } catch {
+                $errorMsg = "Failed to update registry ProfileImagePath: $($_.Exception.Message)"
+                Write-ToLog -Message $errorMsg -Level Error -Step "Revert-Migration"
+                $revertResult.Errors += $errorMsg
+                $revertResult.RegistryUpdated = $false
+            }
+            #endregion Update Registry ProfileImagePath
+
+            #region Restore Profile ACLs
+            if (-not $DryRun) {
+                Write-ToLog -Message "Restoring profile ACLs from backup" -Level Info -Step "Revert-Migration"
+                try {
+                    if ($latestAclBackupFile) {
+                        $backupPath = Join-Path -Path $aclBackupDir -ChildPath $latestAclBackupFile.Name
+                        Restore-ProfileACL -BackupPath $backupPath
+                        Write-ToLog -Message "Successfully restored profile ACLs from: $($latestAclBackupFile.Name)" -Level Info -Step "Revert-Migration"
+                    } else {
+                        Write-ToLog -Message "No ACL backup file found to restore permissions." -Level Warning -Step "Revert-Migration"
+                    }
+                } catch {
+                    $errorMsg = "Failed to restore profile ACLs: $($_.Exception.Message)"
+                    Write-ToLog -Message $errorMsg -Level Error -Step "Revert-Migration"
+                    $revertResult.Errors += $errorMsg
+                }
+            } else {
+                Write-ToLog -Message "WHAT IF: Would restore profile ACLs from backup file: $($latestAclBackupFile.Name)" -Level Verbose -Step "Revert-Migration"
+            }
+            #endregion Restore Profile ACLs
+
+            #region Take Ownership of Profile Directory
+            if (-not $DryRun) {
+                Write-ToLog -Message "Setting ownership of profile directory: $profileImagePath" -Level Verbose -Step "Revert-Migration"
```

Comment:
Whats the point in setting these logs here? I'd prefer to add these to the jcAdmu.log if possible